pkg/services/authz/rbac/service.go

```diff
@@ -224,6 +230,7 @@ func (s *Service) List(ctx context.Context, req *authzv1.ListRequest) (*authzv1.
 	}
 
 	resp, err := s.listPermission(ctx, permissions, listReq)
+	resp.Zookie = &authzv1.Zookie{Timestamp: time.Now().Unix()}
```

Comment:
That should depend on cache hits.
Maybe a good heuristic would be to sub the cache ttl if we hit any of the caches?